packages/web/src/components/TeeSidebar.vue

```diff
@@ -61,40 +54,33 @@
         no-outline
         @click="usedTracksRegrouped[categ][0].id"
       /> -->
-      <div
-        v-for="usedTrack in usedTracksRegrouped[categ]"
-        :key="usedTrack.id">
-        <div
-          class="fr-mb-1v">
+      <div v-for="usedTrack in usedTracksRegrouped[categ]" :key="usedTrack.id">
+        <div class="fr-mb-1v">
           <DsfrButton
-            :label="tracks.getTrackTitle(usedTrack.id, choices.lang)"
+            :label="tracks.getTrackTitle(usedTrack.id as TrackId, choices.lang)"
```

Comment:
note: précédemment j'avais des alertes dans mon IDE quand j'utilisais cette syntaxe pour spécifier les typages. Mais c pe une histoire de config de mon IDE

Reply:
cf: https://vuejs.org/guide/typescript/overview#typescript-in-templates